packages/cbioportal-utils/src/mutation/MutationUtils.ts

```diff
@@ -66,6 +72,73 @@ export function extractGenomicLocation(
     }
 }
 
+export function generateHgvsgByMutation(
+    mutation: Partial<Mutation & { chr?: string }>
+): string | undefined {
+    if (hasValidGenomicLocation(mutation)) {
+        const chromosome = findChromosome(mutation);
+
+        // ins
+        if (mutation.referenceAllele === '-') {
+            return `${chromosome}:g.${mutation.startPosition}_${mutation.endPosition}ins${mutation.variantAllele}`;
+        }
+        // del (in new format that no ref after del, may differ with genome nexus hgvsg response)
```

Comment:
I guess it should be the same format after merging https://github.com/genome-nexus/genome-nexus/pull/466